pkg/setting/setting.go

```diff
@@ -1351,6 +1353,11 @@ func readSecuritySettings(iniFile *ini.File, cfg *Cfg) error {
 	return nil
 }
 
+func readAuthGrafanaComSettings(iniFile *ini.File, cfg *Cfg) {
+	sec := iniFile.Section("auth.grafana_com")
+	cfg.GrafanaComSkipOrgRoleSync = sec.Key("skip_org_role_sync").MustBool(false)
+}
```

Comment:
nit: not necessarily for this PR, but it would be good to have a follow-up refactor to read all Grafana Com settings here.

Reply:
Yup, exactly :)